cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -134,6 +137,22 @@
                     );
                 });
 
+                // prevents single click from messing up the edit call
```

Comment:
we actually removed this once as it creates several complications, for example:

- What if javascript is attached to the link
- What happens when you do CTRL + click

I'm not sure introducing this behaviour will solve our problems